clean_user_registry_secrets/Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+escapism = "*"
+kubernetes = "*"
```

Comment:
can we pin these to avoid surprises?

Reply:
I will pin them but btw they are not pinned in the pipenv file for the flask app. Should I pin there as well? There are quite a few packages that are not pinned there... I am not sure if you would like me to pin them all explicitly based on the lock file.

Reply:
yeah it's a bit sloppy there I agree. I suppose the lock file serves the purpose of "pinning" if we always strictly install with `--deploy` so maybe it's not necessary to pin the versions in the `Pipfile` itself?

Reply:
Yes, with `pipenv` the idea is to only add known version restrictions to the `Pipfile` and leave the "pinning" to the lockfile.

Reply:
Oh ok cool. I will remove the pinning I did in the pipfile then.